acceptance/lib/puppet/acceptance/common_utils.rb

```diff
@@ -1,5 +1,160 @@
 module Puppet
   module Acceptance
+    module BeakerUtils
+      # TODO: This should be added to Beaker
+      def assert_matching_arrays(expected, actual, message = "")
+        assert_equal(expected.sort, actual.sort, message)
+      end
+
+      # TODO: Remove the wrappers to user_present
+      # and user_absent if Beaker::Host's user_present
+      # and user_absent functions are fixed to work with
+      # Arista (EOS).
+
+      def user_present(host, username)
+        case host['platform']
+        when /eos/
+          on(host, "useradd #{username}")
+        else
+          host.user_present(username)
+        end
+      end
+
+      def user_absent(host, username)
+        case host['platform']
+        when /eos/
+          on(host, "userdel #{username}", acceptable_exit_codes: [0, 1])
+        else
+          host.user_absent(username)
+        end
+      end
+    end
+
+    module CronUtils
```

Comment:
The CronUtils modules was removed from 6.0.x in https://github.com/puppetlabs/puppet/commit/0baf9ba68d3192655136f91078542109665f6fba#diff-48f2763d6b2f848734e74f69b76ded23